### DISCUSSION TITLE
How to push translations to production to release a translation update

### DISCUSSION THREAD

**Discussion by toineenzo** [GlotPress/GlotPress-WP#1838] [Q&A] [answered]
We have a plugin that's not in the wp.org store as it is a pro plugin. We have a GlotPress install set up. 
My question is: how do I automatically push the made translations to production, so user receive a translation update inside their WordPress when the pro plugin is installed.
Thanks!

**Comment by pedro-mendonca** [comment, marked as answer]
GlotPress itself doesn't include the Language Pack distribution feature. For w.org there is a custom plugin doing that.
You can do the same for your private repo with https://github.com/wearerequired/traduttore/.

**Reply by toineenzo** [reply]
Thank you, i'll take a look at that!
